tmt/steps/provision/connect.py

```diff
@@ -61,21 +66,21 @@ def options(cls, how=None):
                 help='Password for login into the guest system.'),
             ] + super().options(how)
 
-    def default(self, option, default=None):
+    def default(self, option: str, default: Optional[Any] = None) -> Any:
         """ Return default data for given option """
         # User root as the default user
         if option == 'user':
             return 'root'
         # No other defaults available
         return default
 
-    def wake(self, keys=None, data=None):
+    def wake(self, keys: Optional[List[str]] = None, data: Any = None) -> None:
```

Comment:
Is there better option for `data`?

Reply:
Yep, see: tmt/steps/prepare/ansible.py

Reply:
Fixed

Reply:
it is not that simple, this plugin has different data than the one I referenced ...